client/api/go-client/default_transport_clone_go112.go

```diff
@@ -0,0 +1,43 @@
+// +build !go1.13
+
+//
+// Copyright (c) 2020 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), as published by the Free Software Foundation,
+// or under the Apache License, Version 2.0 <LICENSE-APACHE2 or
+// http://www.apache.org/licenses/LICENSE-2.0>.
+//
+// You may not use this file except in compliance with those terms.
+//
+
+package client
+
+import "net/http"
+
+// make a copy of the default transport at package initialization time
+var defaultTransportCopy = *(http.DefaultTransport.(*http.Transport))
```

Comment:
bah, this fails with:
```
-- Running: 030-govet.sh --
# github.com/heketi/heketi/client/api/go-client
client/api/go-client/default_transport_clone_go112.go:20: variable declaration copies lock value to defaultTransportCopy: net/http.Transport contains sync.Mutex
failed ./tests/030-govet.sh [2]
```

you'll need to do the field-by-field copy here as well

```go
var copyOfDefaultTransport *http.Transport

func init() {
  copyOfDefaultTransport = copyTransport(http.DefaultTransport.(*http.Transport))
}

func copyTransport(in *http.Transport) *http.Transport {
  return &http.Transport{
    Proxy: in.Proxy, 
    ...
  }
}

func defaultTransportClone() *http.Transport {
  return copyTransport(copyOfDefaultTransport)
}
```

Reply:
Good catch, and great presubmit coverage.